site/src/pages/TemplatePage/TemplateEmbedPage/TemplateEmbedPage.tsx

```diff
@@ -144,7 +144,7 @@ export const TemplateEmbedPageView: FC<{
             height={{
               // 80px is the vertical padding of the content area
               // 36px is from the status bar from the bottom
-              md: "calc(100vh - (80px + 36px))",
+              height: "calc(100vh - (80px + 36px))",
```

Comment:
I guess this one couldn't become `100%` for some reason?

Reply:
Ahh, it could!